### PR TITLE
OSS - E1783 Convolutional Data Extraction From GitHub

### PR DESCRIPTION
**Pull request by kaggarw2**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,6 +11,7 @@ gem 'bind-it'
 gem 'capistrano'
 gem "capistrano-db-tasks", require: false
 gem 'capistrano-rails'
+gem 'chartkick'
```
Comment (Winbobob) [inline Gemfile]: You could use `highchart` javascript library, which has already been used in Expertiza.

File: app/views/submission_records/index.html.erb [hunk 2]
```diff
@@ -29,4 +29,23 @@
       </tr>
   <% end %>
   </tbody>
+
+  <%
+     if @authors.nil?
+     else
+  %>
+  <h4> Commits Over Project Timeline </h4>
+  <%= line_chart @authors.map{ |goal|
```
Comment (Winbobob) [inline app/views/submission_records/index.html.erb]: Please move the logic from view to helper.

File: Gemfile (hunk 1 above)
Comment (RickKwok) [reply Gemfile]: I mentioned using `highchart`

**Comment by kaggarw2** [comment]
Sorry, but the team had no prior knowledge on javascript and we had a time constraint as maximum time was invested in setting up the dev environment.
We had mentioned in our early design that we are keen to use chartkick for graphs. 

Sent from Mail for Windows 10

File: Gemfile (hunk 1 above)
Comment (kaggarw2) [reply Gemfile]: Sorry, but the team had no prior knowledge on javascript and we had a time constraint as maximum time was invested in setting up the dev environment.
We had mentioned in our early design that we are keen to use chartkick for graphs.

**Pull request closed without merging**